File: 01_Language/01_Functions/go/src/functions/ctype_upper.go
package functions

func CtypeUpper(s string) bool {
	if len(s) == 0 {
		return false
	}
	for i := 0; i < len(s); i++ {
		codePoint := int(s[i])
		if codePoint >= 65 && codePoint <= 90 {
			continue
		}
		return false
	}
	return true
}

File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/CtypeUpper.java
package com.onevgo.functions;

public class CtypeUpper {
    public static boolean ctypeUpper(String s) {
        if (s == null || s.length() == 0) return false;
        for (int i = 0; i < s.length(); i++) {
            char codePoint = s.charAt(i);
            // A-Z
            if (codePoint >= 65 && codePoint <= 90) {
                continue;
            }
            return false;
        }
        return true;
    }

    public static void main(String[] args) {

    }
}
